spec/formatting.md

```diff
@@ -8,6 +8,15 @@ when formatting a message for display in a user interface, or for some later pro
 The document is part of the MessageFormat 2.0 specification,
 the successor to ICU MessageFormat, henceforth called ICU MessageFormat 1.0.
 
+## Literal Resolution
+
+The resolved value of _text_, _literal_ and _nmtoken_ tokens
+is always a string concatenation of its parts,
+with escape sequences resolving to their escaped characters.
+When a literal value is used as a formatting function argument or option value,
```

Comment:
I'm also not sure whether the word "value" is meant to be syntactic or semantic in "a literal value". It might be clearer (if more verbose) to write:

"When a _text_, _literal_, or _nmtoken_ token is used as a formatting function argument or option value..."

The current wording, if read literally, implies that the requirement to not distinguish between different syntactic constructs with the same meaning *only* implies to literals. Which would be confusing, since I don't think it's possible to write two lexically distinct literals with the same meaning.

Reply:
If #364 is accepted, this should clear up a bit as the ABNF _literal_ becomes the only thing this applies to. But yes, also here should avoid the dread term "value".